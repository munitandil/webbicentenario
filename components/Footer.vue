<template>
  <footer class="footer_dark">
    <div class="top_footer small_pt">
      <div class="col-12">
        <div class="row justify-content-between">
          <div class="col-lg-4 col-md-12">
            <div class="footer_logo">
              <DSILink pCustomLabel=" " pPath="HOME" pTipoLink="I" pTitle="Logo de la municipalidad de tandil"><img
                  alt="logoMunicipalidad" src="@/assets/images/logo_muni_blanco.gif" style="width: 230px" /></DSILink>
            </div>
            <NetworkSocial pUbicacion="FOOTER" />
          </div>
          <!--<div class="col-lg-4 col-md-6 mt-4 mt-lg-0">
            <h6 class="widget_title">Links Internos</h6>
            <ul class="list_none widget_links_style1">
              <li v-for="item in menuFooter" :key="item.id_item">
                <DSILink
                  :pCustomLabel="item.descr"
                  :pCustomLink="item.link"
                  :pTipoLink="item.tipo_link"
                />
              </li>
            </ul>
          </div> -->
          <div>
            <p> </p>
          </div>
          <div class="col-lg-4 col-md-6 mt-4 mt-lg-0">
            <h6 class="widget_title">Contactanos</h6>

            <ul class="contact_info contact_info_light list_none">
              <li>
                <span class="ti-location-pin"></span>
                <address>Municipio de Tandil • Belgrano 485 • Tandil</address>
              </li>
              <li>
                <span class="ti-email"></span>
                <DSILink pCustomLabel="Teléfonos Útiles" pPath="CONTACTO" pTipoLink="I" />
              </li>
            </ul>
          </div>
        </div>
        <!-- <TelefonosImportantes customClass="dsi-telefonos-importante" /> -->
      </div>
    </div>
    <div class="bottom_footer bg-blue">
      <div class="container">
        <div class="row align-items-center">
          <div class="col-md-6">
            <p class="copyright m-md-0 text-center text-md-left">
              © {{ new Date().getFullYear() }} Todos los derechos reservados
              Municipalidad de Tandil.
            </p>
          </div>
          <div class="col-md-6">
            <p class="developer-by m-md-0 text-center text-md-right">
              <a href="#" title="Desarrollado por: Dirección de Sistemas Informáticos - Municipalidad de Tandil">
                Desarrollado por: <img alt="logoDsi" src="@/assets/images/logo_dsi_con_fondo.png"
                  style="width: 100px" />
              </a>
              <!--  <a href="#" title="Desarrollado por: Dirección de Sistemas Informáticos - Municipalidad de Tandil">
              Desarrollado por: <img
                  alt="logoDsi"
                  src="@/assets/images/logo_dsi_bn.png"
                  style="width: 100px"
              />
              </a> -->
            </p>
          </div>
        </div>
      </div>
    </div>
  </footer>
</template>

<script>
import { mapGetters } from "vuex";
import TelefonosImportantes from "@/components/PhoneImportant.vue";
import DSILink from "@/components/DSILink";
import NetworkSocial from "@/components/NetworkSocial.vue";
export default {
  name: "Footer",
  components: { TelefonosImportantes, DSILink, NetworkSocial },
  computed: {
    ...mapGetters({
      menuFooter: "paths/getMenuFooter",
    }),
  },
};
</script>