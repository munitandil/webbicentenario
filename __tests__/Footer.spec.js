import { shallowMount } from "@vue/test-utils";
import Footer from "@/components/Footer";


const factory = () => {
    return shallowMount(Footer, {
        stubs: {
            'nuxt-link': true,
            'any-other-child': true
        }
    });
};

describe("Footer", () => {
    /*test("mounts properly", () => {
        const wrapper = factory();
        expect(wrapper.vm).toBeTruthy();
    }); */

    test("renders properly", () => {
        const wrapper = factory();
        expect(wrapper.html()).toMatchSnapshot();
    });
});