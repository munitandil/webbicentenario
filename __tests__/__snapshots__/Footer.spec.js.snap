// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer mounts properly 1`] = `
<footer class="footer_dark">
  <div class="top_footer">
    <div class="container">
      <div class="row">
        <div data-animation="fadeInUp" data-animation-delay="0.1s" class="col-lg-4 col-md-12 animation">
          <div class="footer_logo">
            <nuxt-link-stub to="/" class="navbar-brand"><img src="~assets/images/construction-logo_white.png" alt="logo" class="logo_light"> <img src="~assets/images/header_logo_mun_96_41.png" alt="logo" class="logo_dark"> <img src="~assets/images/header_logo_mun_96_41.png" alt="logo" class="logo_default"></nuxt-link-stub>
          </div>
          <p>Phasellus blandit massa enim. elit id varius nunc. Lorems ipsum dolor sit amet, consectetur industry.</p>
          <ul class="list_none footer_social">
            <li><a href="#"><i class="ion-social-facebook"></i></a></li>
            <li><a href="#"><i class="ion-social-twitter"></i></a></li>
            <li><a href="#"><i class="ion-social-googleplus"></i></a></li>
            <li><a href="#"><i class="ion-social-youtube-outline"></i></a></li>
            <li><a href="#"><i class="ion-social-instagram-outline"></i></a></li>
          </ul>
        </div>
        <div data-animation="fadeInUp" data-animation-delay="0.2s" class="col-lg-4 col-md-6 mt-4 mt-lg-0 animation">
          <h6 class="widget_title">USEFUL LINKS</h6>
          <ul class="list_none widget_links_style1">
            <li><a href="#">Buy Anger Now</a></li>
            <li><a href="#">About Us</a></li>
            <li><a href="#">Features</a></li>
            <li><a href="#">Feedback</a></li>
            <li><a href="#">Support center</a></li>
            <li><a href="#">Contact Us</a></li>
          </ul>
        </div>
        <div data-animation="fadeInUp" data-animation-delay="0.3s" class="col-lg-4 col-md-6 mt-4 mt-lg-0 animation">
          <h6 class="widget_title">Subscribe Newsletter</h6>
          <p>Contrary to popular belief of lorem Ipsm Latin amet ltin from consectetur industry.</p>
          <div class="newsletter_form">
            <form>
              <div class="radius_input"><input type="text" required="required" placeholder="Enter Email Address"></div> <button type="submit" title="Subscribe" name="submit" value="Submit" class="btn btn-default">
                subscribe
              </button>
            </form>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div class="bottom_footer bg-dark">
    <div class="container">
      <div class="row align-items-center">
        <div class="col-md-6">
          <p class="copyright m-md-0 text-center text-md-left">© 2018 All Rights Reserved by Anger.</p>
        </div>
        <div class="col-md-6">
          <ul class="list_none footer_link text-center text-md-right">
            <li><a href="#">Privacy Policy</a></li>
            <li><a href="#">Terms &amp; Conditions</a></li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</footer>
`;

exports[`Footer renders properly 1`] = `
<footer class="footer_dark">
  <div class="top_footer">
    <div class="container">
      <div class="row">
        <div data-animation="fadeInUp" data-animation-delay="0.1s" class="col-lg-4 col-md-12 animation">
          <div class="footer_logo">
            <nuxt-link-stub to="/" class="navbar-brand"><img src="~assets/images/construction-logo_white.png" alt="logo" class="logo_light"> <img src="~assets/images/header_logo_mun_96_41.png" alt="logo" class="logo_dark"> <img src="~assets/images/header_logo_mun_96_41.png" alt="logo" class="logo_default"></nuxt-link-stub>
          </div>
          <p>Phasellus blandit massa enim. elit id varius nunc. Lorems ipsum dolor sit amet, consectetur industry.</p>
          <ul class="list_none footer_social">
            <li><a href="#"><i class="ion-social-facebook"></i></a></li>
            <li><a href="#"><i class="ion-social-twitter"></i></a></li>
            <li><a href="#"><i class="ion-social-googleplus"></i></a></li>
            <li><a href="#"><i class="ion-social-youtube-outline"></i></a></li>
            <li><a href="#"><i class="ion-social-instagram-outline"></i></a></li>
          </ul>
        </div>
        <div data-animation="fadeInUp" data-animation-delay="0.2s" class="col-lg-4 col-md-6 mt-4 mt-lg-0 animation">
          <h6 class="widget_title">USEFUL LINKS</h6>
          <ul class="list_none widget_links_style1">
            <li><a href="#">Buy Anger Now</a></li>
            <li><a href="#">About Us</a></li>
            <li><a href="#">Features</a></li>
            <li><a href="#">Feedback</a></li>
            <li><a href="#">Support center</a></li>
            <li><a href="#">Contact Us</a></li>
          </ul>
        </div>
        <div data-animation="fadeInUp" data-animation-delay="0.3s" class="col-lg-4 col-md-6 mt-4 mt-lg-0 animation">
          <h6 class="widget_title">Subscribe Newsletter</h6>
          <p>Contrary to popular belief of lorem Ipsm Latin amet ltin from consectetur industry.</p>
          <div class="newsletter_form">
            <form>
              <div class="radius_input"><input type="text" required="required" placeholder="Enter Email Address"></div> <button type="submit" title="Subscribe" name="submit" value="Submit" class="btn btn-default">
                subscribe
              </button>
            </form>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div class="bottom_footer bg-dark">
    <div class="container">
      <div class="row align-items-center">
        <div class="col-md-6">
          <p class="copyright m-md-0 text-center text-md-left">© 2018 All Rights Reserved by Anger.</p>
        </div>
        <div class="col-md-6">
          <ul class="list_none footer_link text-center text-md-right">
            <li><a href="#">Privacy Policy</a></li>
            <li><a href="#">Terms &amp; Conditions</a></li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</footer>
`;
